### PR TITLE
Bug 1801799: openstack docs day 2 ssc refresh

### PR DESCRIPTION
**Pull request by iamemilio**
Documents how to refresh certs post install

### CODE REVIEW

File: docs/user/openstack/README.md [hunk 1]
```diff
@@ -498,6 +498,14 @@ Another useful thing to check is that the ignition configurations are only avail
 curl https://<loadbalancer ip>:22623/config/master --insecure
 ```
 
+## Refreshing a CA Certificate
+
+If you ran the installer with a custom CA certificate, then this certificate can be changed while the installer is running as needed. To chage your certificate, edit the value of the `ca-cert.pem` key in the `cloud-provider-config` configmap with a valid PEM certificate.
```
Comment (tomassedovic) [inline docs/user/openstack/README.md]: The "while the installer is running" phrase makes me think you would want to do this during the installation process.

But I think this is more for updating the cert after the installation finished, during normal operation of the OpenShift cluster? Would you mind changing it to something like "while the cluster is running"?

Sorry if I completely misunderstood what this is for!

File: docs/user/openstack/README.md [hunk 2]
```diff
@@ -498,6 +498,14 @@ Another useful thing to check is that the ignition configurations are only avail
 curl https://<loadbalancer ip>:22623/config/master --insecure
 ```
 
+## Refreshing a CA Certificate
```
Comment (mandre) [inline docs/user/openstack/README.md]: Could you also update the ToC?

File: docs/user/openstack/README.md (hunk 1 above)
Comment (mandre) [inline docs/user/openstack/README.md]: This should link to the [Self Signed OpenStack CA certificates](https://github.com/openshift/installer/blob/83f2681dbeb943fc9fbe44e8ea8514b2e6362d07/docs/user/openstack/README.md#self-signed-openstack-ca-certificates) section.

I wonder if it wouldn't be better to make a new "Post-Deployment" section and move the External Load Balancer and the Self-signed Cert refresh under it?

And a typo and rewording suggestion:
s/chage/change/
s/can be changed while the installer is running as needed/can be changed later on/

Comment (iamemilio) [reply docs/user/openstack/README.md]: Ah good catch, I meant "while the cluster is runnning."

**Comment by sdodson** [comment]
I see that there's a cloned bug for this, generally we don't bother backporting changes to older installer branches. Ultimately this needs to make it into product documentation. Is there a separate PR for that?

File: docs/user/openstack/README.md (hunk 1 above)
Comment (iamemilio) [reply docs/user/openstack/README.md]: I think I prefer "while the cluster is running" just to clarify that this wont blow anything up

**Comment by iamemilio** [comment]
The only reason that I created a backport bugzilla is because we backported into a z stream of 4.3, and this was left out. I informed @maxwelldb (downstream docs) of the issue, so they are aware of the situation.

**Comment by sdodson** [comment]
> The only reason that I created a backport bugzilla is because we backported into a z stream of 4.3, and this was left out.

Ok, as long as it ultimately ends up in product docs that's fine. Sorry for the interjection.

**Comment by tomassedovic** [comment]
/lgtm

Thanks!

**Comment by sdodson** [comment]
/override ci/prow/e2e-aws-upgrade
No point in spinning up clusters for README changes.

**Pull request merged**